media-sound/gnome-music/gnome-music-48.0.ebuild

```diff
@@ -0,0 +1,73 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+PYTHON_COMPAT=( python3_{11..13} )
+
+inherit gnome.org gnome2-utils meson python-single-r1 xdg
+
+DESCRIPTION="Music management for Gnome"
+HOMEPAGE="https://apps.gnome.org/Music/ https://gitlab.gnome.org/GNOME/gnome-music/"
+
+LICENSE="GPL-2+"
+SLOT="0"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~riscv ~x86"
+
+DEPEND="${PYTHON_DEPS}
+	>=dev-libs/glib-2.67.1:2
+	>=dev-libs/gobject-introspection-1.54:=
+	>=gui-libs/gtk-4.10.0:4[introspection]
```

Comment:
Hello,

You can use:
pkgdiff-mg --build-system gnome-music-47.1.ebuild gnome-music-48.0.ebuild

to help you to catch updated deps like, for example, updated gtk version needed for this version

Thanks